lib/grunt/utils.js

```diff
@@ -122,12 +122,12 @@ module.exports = {
 
   process: function(src, NG_VERSION, strict) {
     var processed = src
-      .replace(/"NG_VERSION_FULL"/g, NG_VERSION.full)
-      .replace(/"NG_VERSION_MAJOR"/, NG_VERSION.major)
-      .replace(/"NG_VERSION_MINOR"/, NG_VERSION.minor)
-      .replace(/"NG_VERSION_DOT"/, NG_VERSION.patch)
-      .replace(/"NG_VERSION_CDN"/, NG_VERSION.cdn)
-      .replace(/"NG_VERSION_CODENAME"/, NG_VERSION.codeName);
+      .replace(/(['"])NG_VERSION_FULL\1/g, NG_VERSION.full)
+      .replace(/(['"])NG_VERSION_MAJOR\1/, NG_VERSION.major)
+      .replace(/(['"])NG_VERSION_MINOR\1/, NG_VERSION.minor)
+      .replace(/(['"])NG_VERSION_DOT\1/, NG_VERSION.patch)
+      .replace(/(['"])NG_VERSION_CDN\1/, NG_VERSION.cdn)
```

Comment:
I could find this anywhere inside the codebase. It is probably not needed, but left it just in case.